apollo-api/src/main/java/com/apollographql/apollo/api/ResponseReader.java

```diff
@@ -2,7 +2,9 @@
 
 import java.io.IOException;
 
-/** TODO **/
+/*
+ * ResponseReader is responsible for converting a field object to another object of type T.
+ */
```

Comment:
I would change this to: ResponseReader is an abstraction for reading GraphQL fields.